viewer_clients/web-client/style/codecheckerviewer.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway|Source+Code+Pro');
```

Comment:
How should we handle it then? Should the `build_package.py` also download and "copy" the font into the webserver's folder?

Fonts available on _Google Fonts_ are entirely open-source and free to use even in commercial projects. (Though they advise _against_ downloading and serving it locally, it is not _legally_ prohibited.)